### PR TITLE
KEP-1416 macOS only - flush the database during successful call to create and update, thi…

### PR DESCRIPTION
**Pull request by rnistuk**
…s ensures that the SIGABRT is avoided

### CODE REVIEW

File: storage/rocksdb_storage.cpp [hunk 1]
```diff
@@ -158,6 +171,10 @@ rocksdb_storage::update(const bzn::uuid_t& uuid, const bzn::key_t& key, const bz
         this->update_metadata_size(uuid, NAMESPACE_KEY, SIZE_KEY, ns_prev_size - prev_size + value.size() + key.size());
         this->update_metadata_size(uuid, SIZE_KEY, key, value.size() + key.size());
 
+#ifdef __APPLE__
```
Comment (ebruck) [inline storage/rocksdb_storage.cpp]: perhaps move the flush to the update_metadata_size instead?

Comment (rnistuk) [reply storage/rocksdb_storage.cpp]: That's a good idea.

Comment (ebruck) [reply storage/rocksdb_storage.cpp]: Let's create a backlog item to properly understand what is going on here.

Comment (rnistuk) [reply storage/rocksdb_storage.cpp]: After discussions with Ed, I've decided the leave the flushes where they are and create a backlog item to find the real problem.

**Pull request merged**